api/src/main/java/org/openbaton/nfvo/api/admin/RestVimInstances.java

```diff
@@ -100,6 +98,7 @@ public void delete(
    *
    * @return List<Datacenter>: The List of Datacenters available
    */
+  @ApiOperation(value = "Retrieving all Vim Instances", notes = "")
```

Comment:
"This method returns the list of all the VimInstances on-boarded in this project identified by the header project-id"